docs/release.md

```diff
@@ -1,125 +1,112 @@
 # `taproot-assets`'s Reproducible Build System
 
-This package contains the build script that the `taproot-assets` project uses in
-order to build binaries for each new release. As of `go1.13`, with some new
-build flags, binaries are now reproducible, allowing developers to build the
-binary on distinct machines, and end up with a byte-for-byte identical binary.
-However, this wasn't _fully_ solved in `go1.13`, as the build system still
-includes the directory the binary is built into the binary itself. As a result,
-our scripts utilize a work around needed until `go1.13.2`.  
+Our release artifacts are designed to be reproducible, meaning, for any
+release version:
 
-## Building a New Release
+1) Release binaries produced for any target architecture correspond
+   byte-for-byte, independent of what host machine or architecture was
+   used to perform the build. For example: if a darwin-arm64 machine and
+   a linux-amd64 machine each build release binaries for windows-amd64,
+   then those produced binaries will match exactly.
 
-### MacOS
+2) These binaries, as well as the source and vendored dependencies
+   used to build them, are packaged in archives that can be reproduced
+   exactly.
 
-The first requirement is to have [`docker`](https://www.docker.com/)
-installed locally and running. The second requirement is to have `make`
-installed. Everything else (including `golang`) is included in the release
-helper image.
+In each case, we confirm an "exact match" by comparing SHA256 digests.
 
-To build a release, run the following commands:
+## Building a Release
+
+To build a release, ensure `make` and `git` are installed, and that
+[`Docker`](https://www.docker.com/) is installed locally and running.
+Then:
 
 ```shell
-$  git clone https://github.com/lightninglabs/taproot-assets.git
-$  cd taproot-assets
-$  git checkout <TAG> # <TAG> is the name of the next release/tag
-$  make docker-release tag=<TAG>
+$ git clone https://github.com/lightninglabs/taproot-assets.git
+$ cd taproot-assets
+$ git checkout "$TAG"
+$ make docker-release tag="$TAG"
```

Comment:
I think we should be able to speed up the docker-release (at least on mac) using a similar technique as my recent PR to speed up linting.